src/query/interpreter.cpp

```diff
@@ -2425,11 +2564,25 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
   }
 }
 
+std::vector<std::string> Interpreter::GetQueriesSummary() {
+  if (in_explicit_transaction_) {
+    return {"EXPLICIT TRANSACTION"};
+  }
+  std::vector<std::string> queries;
+  for (const auto &query_exec_ptr : query_executions_) {
+    if (query_exec_ptr) queries.push_back(query_exec_ptr->query);
```

Comment:
should this ever be null?

Reply:
I think yes, but this part needs to be a little bit redesigned I think, stay tuned :smile: